guide/matrix-vs-discord.md

```diff
@@ -35,16 +35,18 @@ I'm sure you can find a lot of valid criticisms of Discord, like [here](https://
 
 In the scope of this guide, the key reasons to move from Discord to Matrix are:
 
-* **Lack of privacy for private communications**, as private conversations are not only unencrypted, but also actively scanned (see your settings).
-* **Excessive tracking that cannot be opted-out**, such as the [science endpoint](https://luna.gitlab.io/discord-unofficial-docs/science.html) and the process logger (for activity status).
-* **Hostile stance against unofficial clients or client modifications**, thus preventing users from opting out of certain annoyances or tracking.
-* **Arbitrary phone number requirements**.
+* **Lack of privacy for private communications**, as private conversations are not only unencrypted, but also actively scanned (scope of which depends on your settings, but some happen even when you disable all the filtering-related toggles). People reported their messages were substituted with cooking recipes when they tried to send something that is considered a virus by Discord.
```

Comment:
> People reported their messages were substituted with cooking recipes when they tried to send something that is considered a virus by Discord.

source

Reply:
https://twitter.com/Ashfire908/status/1021266005664780289 reports a bug on that very error message.

Reply:
I don't think the messages are substituted, and also it's quite old... Is that still the case now?

Reply:
Yes, it still is. My friends have tried it a few weeks ago and it is still the case. And guess what, links in those substituted messages broke again...

Reply:
> I don't think the messages are substituted, and also it's quite old... Is that still the case now?

I have personally seen it happen so i can confirm it does happen.